src/README.rst

```diff
@@ -19,6 +19,7 @@ Change Log
 Unreleased
 -----
 - Add application health policies parameter for config upgrade, which are available in Service Fabric runtime 6.3 (#92)
+- Add command to see the current version and target azure-servicefabric version (#104)
```

Comment:
nit: "target azure-servicefabric version" -> "the target azure-servicefabric version"

Reply:
We display the target version in -h, so would we need to show it somewhere else as a whole other command?

Reply:
You are saying we should drop the target version from --version? I'm fine with that

Reply:
Yes